src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -4181,6 +4181,9 @@
   <data name="InvalidOperation_ComInteropRequireComWrapperInstance" xml:space="preserve">
     <value>COM Interop requires ComWrapper instance registered for marshalling.</value>
   </data>
+  <data name="InvalidOperation_ComInteropRequireComWrapperTrackerInstance" xml:space="preserve">
+    <value>COM Interop requires ComWrapper instance registered for reference tracker support.</value>
```

Comment:
I do not see this message in CoreCLR anywhere. Is the CoreCLR error handling different? Do we want to make it the same?

Reply:
CoreCLR doesn't have a message, but following the code path, it does throw an ArgumentException in one case and in the other it seems it would hit a NullPointerException from a null access.  Given in CoreCLR they are mixing between C# and the C++ layer for where this logic is and the error happens in a more generic function like CallReleaseObjects, I guess they didn't have a specific error message there.  But here we are able to.

Reply:
@AaronRobinsonMSFT Should we have a test for these paths?

Reply:
Sure. This code path in CoreCLR is a bit more contrived given the set up, but we can construct a test. Due to how the system is built up it is going to be some effort though.